src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+        FileWriterClass.writeToFile();
+    }
+
+    private static void printMarking(int i) {
+        System.out.println(String.format(" [%s] [%s] %s",
+                tasksList.get(i).getTypeIcon(), tasksList.get(i).getStatusIcon(), tasksList.get(i).getDescription()));
+    }
+
+    private static void addMark(String remainingWords) {
+        System.out.println("You are crushing it,1 task down!");
+        Task.markAsDone(tasksList.get(Integer.parseInt(remainingWords) - 1));
+        int i = Integer.parseInt(remainingWords) - 1;
+        printMarking(i);
+        FileWriterClass.writeToFile();
+    }
+
+    private static void unMark(String remainingWords) {
+        System.out.println("I have unchecked it for you");
+        Task.markAsNotDone(tasksList.get(Integer.parseInt(remainingWords) - 1));
+        int i = Integer.parseInt(remainingWords) - 1;
+        printMarking(i);
+        FileWriterClass.writeToFile();
+    }
+
+    private static void deleteTasks(String remainingWords) {
+        System.out.println("Noted. I've removed this task:");
+        int i = Integer.parseInt(remainingWords) - 1;
+        printMarking(i);
+        System.out.println("Now you have " + tasksList.size() + " tasks left");
+        tasksList.remove(i);
+        FileWriterClass.writeToFile();
+
+    }
+
+    private static void printIntro() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello I am Duke your personal life assistant \n" + logo);
+        System.out.println("What can I do for you today?");
+    }
+
+    private static void processList() {
+        if (tasksList.size() == 0) {
+            System.out.println("Nothing to do for now!Take a break!");
+        } else {
+            System.out.println("Here's your plan for a productive day!");
+        }
+        for (int i = 0; i < tasksList.size(); i++) {
+            System.out.println(String.format("%d.[%s] [%s] %s ", i + 1,
+                    tasksList.get(i).getTypeIcon(), tasksList.get(i).getStatusIcon(), tasksList.get(i).getDescription()
+            ));
+        }
+    }
+
+    static void validateInput(String input[]) throws DukeException.InvalidInputException,
+            DukeException.IncompleteInputException {
+
+        if ((!Objects.equals(input[0], "bye"))
+                && (!Objects.equals(input[0], "list"))
+                && (!Objects.equals(input[0], "todo"))
+                && (!Objects.equals(input[0], "event"))
+                && (!Objects.equals(input[0], "deadline"))
+                && (!Objects.equals(input[0], "mark"))
+                && (!Objects.equals(input[0], "unmark"))
+                && (!Objects.equals(input[0], "delete"))) {
```

Comment:
Since these are Strings, perhaps you can use the .equals method from the String class rather than the one from the Objects class